tasks/send-email-alerts-by-event-topic.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Send email alerts by event topic",
+  "options": {
+    "emails_to_notify__array_required": [
+      "brad@tekha.us"
```

Comment:
Leave this out of the task defaults; just use `null` here instead, being really explicit about the fact that you're not providing a default value, but that you are defining the presence of the option.

Reply:
Doh! I'm so used to running a diff against prior task code that I forgot to clean this up.